operator/cmd/mesh/install.go

```diff
@@ -182,6 +190,19 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
 
+	// Workaround for broken validation with revisions (https://github.com/istio/istio/issues/28880)
+	// TODO(Monkeyanator) remove once we have a nicer solution
+	if iArgs.revision != "" {
+		if requiresIstiodServiceCreation(clientset, name.IstioDefaultNamespace) {
+			if err := createIstiodService(clientset, name.IstioDefaultNamespace); err != nil {
+				const canaryUpgradeDoc = "https://istio.io/latest/docs/setup/upgrade/canary/"
```

Comment:
Can you add this URL in [url.go ](https://github.com/istio/istio/blob/master/pkg/url/url.go)package like [this](https://github.com/istio/istio/pull/28701/files#diff-0efc7b25e81850d4fb2f694dab07bbf4c8b7722538b21170b43dbe8f861918c9R60), and use `url.CanaryUpgrades` here. This will make the URL backward compatible and will work with older docs eg: for `1.8`,`1.9`.

Reply:
Fixed, thanks